test/transition.js

```diff
@@ -3,20 +3,15 @@ var transition = require('../src/transition')
 var props = transition.properties;
 
 describe('Transition helpers', () => {
-
-  //beforeEach(()=>{
-  //   document.body.innerHTML = window.__html__['test/fixtures/event.html']
-  // })
-
   it('should parse duration from node property', () => {
     var el = document.createElement('div');
 
-    el.style[props.duration] = '1.4s'
+    el.style[props.duration] = '2s'
 
-    expect(transition.end._parseDuration(el)).to.equal(1400)
+    expect(transition.end._parseDuration(el)).to.equal(2000)
```

Comment:
> I just ran the test with the original inputs. Nothing seems to be broekn when I run it.
 
Ohh, it could by my version of Chrome then. I'm reverting.
(I'm using 53.0.2785.116)